src/main/java/com/hillel/items_exchange/security/jwt/JwtAuthenticationEntryPoint.java

```diff
@@ -32,11 +26,10 @@ public void commence(HttpServletRequest httpServletRequest,
                          HttpServletResponse httpServletResponse,
                          AuthenticationException e) throws IOException {
 
-        try (ServletServerHttpResponse res = new ServletServerHttpResponse(httpServletResponse)) {
-            res.setStatusCode(HttpStatus.UNAUTHORIZED);
-            res.getServletResponse().setHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE);
-            res.getBody().write(mapper.writeValueAsString(
-                    messageSource.getMessage("invalid.token", null, Locale.US)).getBytes());
-        }
+        final String detailedError = (String) httpServletRequest.getAttribute("detailedError");
+
+        httpServletResponse.sendError(HttpServletResponse.SC_UNAUTHORIZED,
+                Objects.requireNonNullElseGet(detailedError, () ->
+                        messageSource.getMessage("invalid.token", null, Locale.US)));
```

Comment:
You should set here ``Locale.getDefault()``. It will help us with sending localized messages in the future.